google-cloud-bigquery/pom.xml

```diff
@@ -12,7 +12,7 @@
   <parent>
     <groupId>com.google.cloud</groupId>
     <artifactId>google-cloud-pom</artifactId>
-    <version>0.8.1-alpha</version>
+    <version>0.8.2-alpha-SNAPSHOT</version>
```

Comment:
Yes - but I don't think Maven Central will recognize -alpha-SNAPSHOT correctly as it doesn't match the standard.